dj/sql/parsing/ast.py

```diff
@@ -671,6 +699,16 @@ def expression(self) -> Optional[Expression]:
         """
         return self._expression
 
+    def namespace_table(self):
+        """
+        Turns the column's namespace into a table.
+        """
+        if self.name.namespace:
+            self._table = Table(name=self.name.namespace)
```

Comment:
Does this need to look for an already existing table with the same name on the ast first? Although I suppose it doesn't matter if there's `_consolidate_duplicates`